lib/tasks/works.rake

```diff
@@ -58,4 +58,27 @@ namespace :works do
       work.destroy
     end
   end
+
+  # See https://github.com/pulibrary/pdc_describe/blob/main/docs/test_data_in_production.md for
+  # more information.
+  desc "Imports works from JSON data"
+  task :import_works, [:path] => :environment do |_, args|
+    if args[:path].blank?
+      puts "Usage: bundle exec rake works:import_works\\[path_to_json_files]"
+      exit 1
+    end
+    path = File.join(args[:path], "*.json")
+    approver = User.first
+    puts "Importing files from: #{path}"
+    Dir.glob(path).each do |file_name|
+      hash = JSON.parse(File.read(file_name))
+      resource = PDCMetadata::Resource.new_from_json(hash)
+      work = Work.new(resource: resource)
+      work.collection = Collection.research_data
+      work.created_by_user_id = approver.id
+      work.state = "approved"
```

Comment:
Set to approved for now because that is what I need to get the records to show in the RSS feed but we could tweak this.